apps/froussard/README.md

```
@@ -0,0 +1,133 @@
+```js
+function handleRequest(context) {
+  const log = context.log;
+  log.info(context.httpVersion);
+  log.info(context.method); // the HTTP request method (only GET or POST supported)
+  log.info(context.query); // if query parameters are provided in a GET request
+  log.info(context.body); // contains the request body for a POST request
+  log.info(context.headers); // all HTTP headers sent with the event
+}
+```
+
+The health checks can be accessed in your browser at [http://localhost:8080/health/readiness]() and [http://localhost:8080/health/liveness](). You can use `curl` to `POST` an event to the function endpoint:
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)